docs/promscale_alerting.md

```diff
@@ -0,0 +1,218 @@
+# Promscale Alerting
+
+This file contains alerting rules which are symptom based. We recommend monitoring your Promscale deployments
+and include the following rules as part of your alerting environment.
+
+## Rules
+
+Copy the below in a `promscale_alerts.yaml` file.
```

Comment:
This is all I need to do? :smile:

Reply:
yeah, i forgot to write adding this to prometheus.yml. Adding that.